packages/esm-service-queues-app/src/queue-table/queue-entry-actions/end-queue-entry.modal.tsx

```diff
@@ -11,27 +11,31 @@ interface EndQueueEntryModalProps {
 
 const EndQueueEntryModal: React.FC<EndQueueEntryModalProps> = ({ queueEntry, closeModal }) => {
   const { t } = useTranslation();
-
+  const patient = queueEntry.display;
+  const queue = queueEntry.queue.display;
   const modalInstruction = (
-    <p>{t('confirmRemovePatientFromQueue', 'Are you sure you want to remove this patient from this queue?')}</p>
+    <Trans i18nKey="confirmRemovePatientFromQueue">
+      Are you sure you want to remove <strong>{{ patient } as any}</strong> from {{ queue }}?
+    </Trans>
   );
 
   return (
     <QueueEntryConfirmActionModal
       queueEntry={queueEntry}
       closeModal={closeModal}
       modalParams={{
-        modalTitle: t('removePatientFromQueue', 'Remove patient from queue'),
+        modalTitle: t('removePatientFromQueue', 'Remove patient from queue?'),
         modalInstruction,
-        submitButtonText: t('removePatient', 'Remove patient'),
+        submitButtonText: t('removePatient', 'Remove'),
```

Comment:
I think we should update the translation key to "remove":

```suggestion
        submitButtonText: t('remove', 'Remove'),
```

Reply:
Great minds, lol